meson.build

```diff
@@ -205,7 +206,18 @@ endforeach
 # Set excutable rpaths for non standard prefixes
 
 if get_option('set_rpath').enabled() or (get_option('set_rpath').auto() and get_option('prefix') != '/usr')
-    install_rpath = get_option('prefix') / get_option('libdir')
+    if host_machine.system() == 'darwin' and get_option('build_macos_app').enabled()
+        app_path = get_option('prefix')
+        app_contents = app_path / 'Contents'
+
+        install_rpath = app_path
+        install_libs_rpath = app_contents / 'Resources' / 'lib'
+        install_headers_rpath = install_libs_rpath
+    else
+        install_rpath = get_option('prefix') / get_option('libdir')
+        install_libs_rpath = install_rpath
+        install_headers_rpath = install_rpath
```

Comment:
I don't think there is such a thing as a `rpath` for header files and I would like to keep this section closer to the original, if possible. Couldn't we also fix the library path as part of the `meson setup` call? Something like: `meson setup --libdir=Contents/Resources/lib ...`.

Reply:
> I didn't find a way to set header files path in `meson setup`. If I set the path by using `meson setup --libdir`, I can only install it under the `gtkwave.app/include`folder, but I actually want to install it under `gktwave.app/Contents/Resources/include`

Did you try `meson configure --includedir=Contents/Resources/include`?